docs/guide/experimental-apis.md

```diff
@@ -1,117 +0,0 @@
-# Experimental APIs
```

Comment:
Was this included in this PR by mistake? It seems like it should have been in the other two.

Reply:
I decided to include it here, which I suppose is a bit kooky, but it should all make sense once the other two are merged in 😅